### ISSUE TITLE
very slow navigation with arrows, auto height, and problem with centering

### ISSUE THREAD

**Issue by lironrichkid** [smashingboxes/OwlCarousel2#849]
I have few problems with your video carousel.
this is the website:
http://www.meonthemic.co.il/he/company/a/videos/

1) you can see that the 2 arrows next and prev are very very slow. owl-next,owl-prev 3 seconds or even more in mobile phone to move the carousel why? what is the bug.
2) i should call when body onload to the   window.dispatchEvent(new Event('resize'));
for change the play location. as you can see when i open it first in a very narrow screen it is not centered and only after resizing this is better. have i call that command or may there is another way?
3) maybe the 300 height isnt good for small screen why isnt it just responsive?
4)why when i click on play the after div is disappear?(http://www.meonthemic.co.il/he/000Frames/site/images/tv-bottom.png)
maybe there is a way to make it look better move faster?
5)while a video is playing there is orange border on it and you cant touch it to make it move. why?

thanks a lot

**Comment by iandonovan** [comment]
Please read our [bug report guide](https://github.com/smashingboxes/OwlCarousel2/#bug-reports) and submit a new issue for each point in your post.

**Comment by lironrichkid** [comment]
the link isnt working

**Comment by lironrichkid** [comment]
@iandonovan

**Comment by greg5green** [comment]
The [bug report guide](https://github.com/smashingboxes/OwlCarousel2/blob/develop/CONTRIBUTING.md#bug-reports) was moved. https://github.com/smashingboxes/OwlCarousel2/blob/develop/CONTRIBUTING.md#bug-reports
